src/main/java/impl/ReaderFromFileImpl.java

```diff
@@ -0,0 +1,18 @@
+package impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import service.ReaderFromFile;
+
+public class ReaderFromFileImpl implements ReaderFromFile {
+    @Override
+    public List<String> read(String fromFileName) {
```

Comment:
The method parameter `fromFileName` should not be hardcoded. It is good that you are passing the file name as a parameter, but make sure that the file path is not hardcoded elsewhere in the application.